tfutils/base.py

```diff
@@ -216,17 +242,34 @@ def __init__(self,
         self.conn = pymongo.MongoClient(host=self.host, port=self.port)
         self.conn.server_info()
         self.collfs = gridfs.GridFS(self.conn[self.dbname], self.collname)
-        recent_name = '_'.join([self.dbname, self.collname, self.exp_id, '__RECENT'])
+
+        if 'port_rec' in save_params:
```

Comment:
Feel free to delete every thing related to "port_rec", it's used for sending another copy of records to another database, trying to overcome the problem that the database was too slow at that time